ui/app/AppLayouts/Communities/panels/SharedAddressesPanel.qml

```diff
@@ -164,8 +165,8 @@ Control {
 
         readonly property var shareAddressesButton: StatusButton {
             visible: !root.isEditMode
-            interactive: root.eligibleToJoinAs !== PermissionTypes.Type.None && root.joinPermissionsCheckSuccessful
-            loading: root.requirementsCheckPending
+            interactive: !root.checkingPermissionToJoinInProgress && root.eligibleToJoinAs !== PermissionTypes.Type.None && root.joinPermissionsCheckSuccessful
```

Comment:
Can you also rename `joinPermissionsCheckSuccessful` to `joinPermissionsCheckCompled` to avoid teh confusion I had the last time :smile:

Reply:
I can't :) We know when it completed (that's the property I just introduced here in this PR: `checkingPermissionToJoinInProgress`)

What `joinPermissionsCheckSuccessful` is that whether the call completed successfully

Reply:
Maybe then `joinPermissionsCheckCompletedWithoutErrors`? Because it's really easy to be confused with the current naming. That property only means whether or not the call finished correctly, but the criteria might still be unsatisfied.

Reply:
> Maybe then `joinPermissionsCheckCompletedWithoutErrors`? Because it's really easy to be confused with the current naming. That property only means whether or not the call finished correctly, but the criteria might still be unsatisfied.

Yeah... that's a good idea/suggestion; I'll do that

Reply:
Fixed, renamed :)